superset-frontend/src/dashboard/components/nativeFilters/FilterBar/CrossFilters/VerticalCollapse.tsx

```diff
@@ -70,32 +46,32 @@ const CrossFiltersVerticalCollapse = (props: {
   }
 
   return (
-    <StyledCollapse
+    <Collapse
       ghost
       defaultActiveKey="crossFilters"
       expandIconPosition="right"
-    >
-      <Collapse.Panel
-        key="crossFilters"
-        header={
-          <StyledCrossFiltersTitle>
-            {t('Cross-filters')}
-          </StyledCrossFiltersTitle>
-        }
-      >
-        {crossFiltersIndicators}
-        <span
-          data-test="cross-filters-divider"
-          css={css`
-            width: 100%;
-            height: 1px;
-            display: block;
-            background: ${theme.colors.grayscale.light3};
-            margin: ${theme.sizeUnit * 8}px auto 0 auto;
-          `}
-        />
-      </Collapse.Panel>
-    </StyledCollapse>
+      items={[
+        {
+          key: 'crossFilters',
+          label: t('Cross-filters'),
+          children: (
+            <>
+              {crossFiltersIndicators}
+              <span
+                data-test="cross-filters-divider"
+                css={css`
+                  width: 100%;
+                  height: 1px;
+                  display: block;
+                  background: ${theme.colors.grayscale.light3};
```

Comment:
Do we need this? Can we use an Ant Design color instead? Also, this should be a standard atomic component?